### DISCUSSION TITLE
'🔍' not supported in a workflow_run trigger?

### DISCUSSION THREAD

**Discussion by vadi2** [community/community#50835] [Actions] [answered]
### Select Topic Area

Question

### Body

I've got an action with the following - `name: "🔍 Check improvements with Mudlet's C++ style guide"` which has been working out pretty great for a number of years. Recently I added a follow-up action that depends on this action for completion with `workflows: ["🔍 Check improvements with Mudlet's C++ style guide"]`, however Github didn't seem to like it:

![image](https://user-images.githubusercontent.com/110988/227268581-bfb5c5a9-513c-4d7e-9321-2a646ebb9176.png)

![image](https://user-images.githubusercontent.com/110988/227268562-d6dc1b23-4946-4ca1-a522-2ab6fa41bca5.png)

Is this intended?

**Comment by vadi2** [comment]
The emoji doesn't seem to be the problem, as it it removed, it is [still an issue](https://github.com/Mudlet/Mudlet/actions/runs/4515667367):

> Encountered an issue parsing workflow trigger(s) "Check improvements with C++ style guide" in a workflow. Please ensure any special characters are escaped.

**Comment by ChristopherHX** [comment, marked as answer]
Have you tried to escape all special glob chars of your workflow name? (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)
```
workflows: ["🔍 Check improvements with Mudlet's C\\+\\+ style guide"]
```

_idk if this is your problem here, but I have not tested it myself. I'm pretty shure `\` allows you to escape any glob char, even if the GitHub Docu doesn't mention it_

**Reply by vadi2** [reply]
That must be it, because removing `+` also did the job.

**Comment by ss28041981ss-eng** [comment]
![Uploading IMG_20251028_215015.jpg…]()
